api/main_endpoints/routes/Messages.js

```diff
@@ -180,17 +188,30 @@ router.get('/listen', async (req, res) => {
 
       if(!clients[id]){
         clients[id] = [];
+
+        // increase total amount of rooms opened
+        MetricsHandler.totalRoomsOpened.inc();
+
+        // increase the current rooms open gauge
+        MetricsHandler.currentRoomsOpen.inc();
       }
 
+      // add connection to the connections open gauge
+      MetricsHandler.currentConnectionsOpen.inc();
```

Comment:
how about instead of current and total rooms open, we just use this metric

`currentConnectionsOpen` can have a label, `roomId` so its easy to know how many connections are open and for what room